### PR TITLE
Adding package OceansDB

### PR DESCRIPTION
**Pull request by castelao**
This package depends on #12320 

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by castelao** [comment]
The missing dependency that caused this failure was included in conda-forge. How could I trigger the checks here to run again? Thanks!

**Comment by scopatz** [comment]
Push a new commit

**Comment by scopatz** [comment]
`git commit --allow-empty -am "trigger rebuild"`

File: recipes/OceansDB/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "OceansDB" %}
+{% set version = "0.8.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (luizirber) [inline recipes/OceansDB/meta.yaml:9]:
```suggestion
  url: "https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz"
```

**Pull request merged**